runtime/near-vm-runner/src/tests/contract_preload.rs

```diff
@@ -1,3 +1,6 @@
+// Currently only testing wasmer code, so disabled outside of x86_64
+#![cfg_attr(not(target_arch = "x86_64"), ignore)]
```

Comment:
Hm, `ignore` isn't supported for modules, no?

Reply:
Yes, this doesn't seem to work at all.

Reply:
Oh right. @nagisa I'm cancelling the changes relating to https://github.com/near/nearcore/pull/6200#discussion_r793866215 ; as the file doesn't compile without wasmer at all (because the things it tests don't compile without wasmer)